scripts/translator.js

```diff
@@ -0,0 +1,47 @@
+import morseCode from './morse-characters.js';
+
+// Keys are morse code, and values arecorresponding english
+const englishFromMorse = Object.entries(morseCode).reduce((acc, [key, value]) => {
+  acc[value] = key;
+  return acc;
+}, {});
+
+// English to morse code
+export function translateToMorse(english) {
+  const result = [];
+  // go over each character
+  for (let char of english.toUpperCase()) {
+    if (char === ' ') {
```

Comment:
you can put an empty space as a key in your dictionary and save yourself the if else check

Reply:
```js 
{ 
   a: 'alex',
   'b': 'bob',
   " ": "/"
} 
```